sources/functions/color_echo

```diff
@@ -60,19 +53,22 @@ echo_error () {
 }
 export -f echo_error
 
-
 echo_info () {
     colorprint "${bold}INFO" "$1"
 }
 export -f echo_info
 
+echo_docs () {
+    colorprint "${cyan}${bold}DOCS" "$1"
```

Comment:
I'd change this to `colorprint "${cyan}${bold}DOCS" "${italic}Read more here: ${under}https://swizzin.ltd/$1"` so we can just pass `echo_docs "applications/jellyfin"` to it.